uncompyle6/scanners/scanner3.py

```diff
@@ -271,20 +274,37 @@ def ingest(self, co, classname=None, code_objects={}, show_asm=None):
                     pattr = const
                     pass
             elif opname in ('MAKE_FUNCTION', 'MAKE_CLOSURE'):
-                pos_args, name_pair_args, annotate_args = parse_fn_counts(inst.argval)
-                if name_pair_args > 0:
-                    opname = '%s_N%d' % (opname, name_pair_args)
-                    pass
-                if annotate_args > 0:
-                    opname = '%s_A_%d' % (opname, annotate_args)
-                    pass
-                opname = '%s_%d' % (opname, pos_args)
-                pattr = ("%d positional, %d keyword pair, %d annotated" %
-                             (pos_args, name_pair_args, annotate_args))
+                if self.version >= 3.6:
+                    # 3.6+ doesn't have MAKE_CLOSURE, so opname == 'MAKE_CLOSURE'
```

Comment:
I know this is minor but I think you mean "opname == 'MAKE_FUNCTION'"? Or did I misunderstand what's going on here?